### PR TITLE
Branch1

### PR DESCRIPTION
**Pull request by jyothishwisilica**
Pull request #1

### CODE REVIEW

File: wiseconnectSuite/wiseconnect/src/main/java/com/wisilica/wiseconnect/WiseconnectSecurityConfiguration.java [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 	@Override
 	public void configure(HttpSecurity http) throws Exception {
-		//http.authorizeRequests().antMatchers("/public/").permitAll();
-		//http.cors().and().csrf().disable();
```
Comment (sreesiv) [inline wiseconnectSuite/wiseconnect/src/main/java/com/wisilica/wiseconnect/WiseconnectSecurityConfiguration.java]: Uncomment this line.

**Review by sreesiv — CHANGES_REQUESTED** [review]

**Review by jyothishwisilica — COMMENTED** [review]
Did as sreesiv suggested.

**Review by sreesiv — APPROVED** [review]

File: wiseconnectSuite/wiseconnect/src/main/java/com/wisilica/wiseconnect/WiseconnectSecurityConfiguration.java [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 	@Override
 	public void configure(HttpSecurity http) throws Exception {
-		//http.authorizeRequests().antMatchers("/public/").permitAll();
+		http.authorizeRequests().antMatchers("/public/").permitAll();
 		//http.cors().and().csrf().disable();
```
Comment (sreesiv) [inline wiseconnectSuite/wiseconnect/src/main/java/com/wisilica/wiseconnect/WiseconnectSecurityConfiguration.java:15]: I am sorry, comment it back.

**Review by sreesiv — CHANGES_REQUESTED** [review]
Comment it back.

File: wiseconnectSuite/wiseconnect/src/main/java/com/wisilica/wiseconnect/WiseconnectSecurityConfiguration.java (hunk 2 above)
Comment (jyothishwisilica) [reply wiseconnectSuite/wiseconnect/src/main/java/com/wisilica/wiseconnect/WiseconnectSecurityConfiguration.java:15]: ok
```suggestion
		//http.cors().and().csrf().disable();
```done.

**Review by jyothishwisilica — COMMENTED** [review]
Done as per sreesiv's 2nd suggestion.

**Review by sreesiv — APPROVED** [review]
Approved.

**Comment by sreesiv** [comment]
Good to go.

**Pull request merged**